semopenalex-app/config/ui.prop

```diff
@@ -1,4 +1,4 @@
 preferredLabels = rdfs:label,skos:prefLabel,<http://purl.org/dc/terms/title>,<http://xmlns.com/foaf/0.1/name>
 showClipboard = true
 preferredThumbnails = { \n     ?subject <http://xmlns.com/foaf/0.1/depiction> ?1 \n    BIND ( replace(STR(?1)\\, "%28"\\, "("\\, "i") as ?2)\n    BIND ( replace(STR(?2)\\, "%29"\\, ")"\\, "i") as ?3)\n    BIND ( replace(STR(?3)\\, "%2C"\\, "\\,"\\, "i") as ?4)\n    BIND ( replace(STR(?4)\\, "%26"\\, "&"\\, "i") as ?5)\n    BIND ( replace(STR(?5)\\, "%"\\, ""\\, "i") as ?6)\n    \n    BIND (IRI(?6) as ?value)\n    \n},<http://schema.org/thumbnail>
-preferredDescriptions = <http://purl.org/dc/terms/abstract>,<http://www.w3.org/2000/01/rdf-schema#comment>,<http://schema.org/description>
+preferredDescriptions = <http://purl.org/dc/terms/abstract>,<http://www.w3.org/2004/02/skos/core#note>,<http://xmlns.com/foaf/0.1/homepage>,<http://www.w3.org/ns/org#memberOf>/<http://xmlns.com/foaf/0.1/name>,<https://dbpedia.org/ontology/location>/<http://www.geonames.org/ontology#countryCode>,<https://semopenalex.org/property/sourceType>
```

Comment:
quite a few properties for description. I think this needs to be changed in the future and fine-tuned. Especially homepage is a strange property for description